spec/unit/util/package/version/rpm_spec.rb

```diff
@@ -0,0 +1,103 @@
+require 'spec_helper'
+require 'puppet/util/package/version/rpm'
+
+describe Puppet::Util::Package::Version::Rpm do
+
+  RpmVersion = Puppet::Util::Package::Version::Rpm
+
+  context "when parsing an invalid version" do
+    it "if is parsing symbols" do
+      expect { RpmVersion.parse(:absent)}.to raise_error
+    end
+  end
+
+  context "when creating new version" do
+    it "is parsing basic version" do
+      v = RpmVersion.parse('1:2.8.8-1.el6')
+      expect(v.epoch).to eql('1')
+      expect(v.version).to eql('2.8.8')
+      expect(v.release).to eql('1.el6')
+      expect(v.arch).to eql(nil)
```

Comment:
As a wise @gimmyxd once told me: "It's not ok to have multiple `expect`s in the same test." 
If the first `expect` fails, you would need to fix it before seeing if any of the others also fail. It's better to have multiple tests with just one `expect` and a friendly name than a test with a generic name and many `expect`s that will end up being frustrating and time consuming when fixing is needed.